### PR TITLE
#166136651  Users should be able to delete articles

### PR DESCRIPTION
**Pull request by vic3king**
#### Description
To provide users more control over the articles the either publish or save as draft. This PR implements the functionality for users to delete their articles

#### Type of change

- [x] New Feature
- [x]

#### How Has This Been Tested?

- [x] Unit testing

#### Checklist:
- [x] create actions and reducers for the route
- [x] add confirmation modal 
- [x] add delete functionality
- [x]  write tests

#### PT-ID

[#166136651 ](https://www.pivotaltracker.com/story/show/166136651)

### CODE REVIEW

**Review by sojida — APPROVED** [review]

File: src/components/containers/profile.container.js [hunk 1]
```diff
@@ -1,16 +1,17 @@
 import { connect } from 'react-redux';
 import Profilepage from '../presentations/Profilepage/ProfilePage';
 import profileAction from '../../actions/profile.action';
-import fetchArticles from '../../actions/articles-update.actions';
 import followeeAction from '../../actions/followee.actions';
 import followingAction from '../../actions/following.actions';
 import fetchBookmarks from '../../actions/bookmarked.action';
 import RequestAction from '../../actions/reviewers-request.action';
+import myArticlesAction from '../../actions/articles-update.actions';
```
Comment (walsamlee) [inline src/components/containers/profile.container.js]: A better name for this variable would be nice.

File: src/components/presentations/Profilepage/ProfilePage.jsx [hunk 2]
```diff
@@ -57,6 +58,12 @@ class Profilepage extends Component {
     followUser(e.target.id, newcount);
   };
 
+  deleteArticleClick = async e => {
```
Comment (walsamlee) [inline src/components/presentations/Profilepage/ProfilePage.jsx:61]: A function name change here to might be needed.

File: src/components/containers/profile.container.js (hunk 1 above)
Comment (vic3king) [reply src/components/containers/profile.container.js]: Like `userArticle` ?

File: src/components/shared/Modals/modal.scss [hunk 3]
```diff
@@ -2,13 +2,13 @@
 
 .modal {
     position: fixed;
-    z-index: 1;
+    // z-index: 1;
```
Comment (walsamlee) [inline src/components/shared/Modals/modal.scss]: Please remove commented style.

File: src/components/shared/Modals/modal.scss [hunk 4]
```diff
@@ -2,13 +2,13 @@
...
     padding-top: 15%;
     left: 0;
     top: 0;
     width: 100%;
     height: 100%;
-    overflow: auto;
+    // overflow: auto;
```
Comment (walsamlee) [inline src/components/shared/Modals/modal.scss]: Remove this too.

File: src/components/containers/profile.container.js (hunk 1 above)
Comment (walsamlee) [reply src/components/containers/profile.container.js]: Yes, `userArticle` is a more suitable variable name.

**Review by McHardex — APPROVED** [review]
looks nice.. nice job

**Review by AnayoOleru — APPROVED** [review]

File: src/components/shared/Modals/modal.scss (hunk 4 above)
Comment (vic3king) [reply src/components/shared/Modals/modal.scss]: alldone @walsamlee

**Pull request merged**